### PR TITLE
Appveyor Implementation

### PR DESCRIPTION
**Pull request by blink1073**
Builds upon #1057.  Corrected dependency installation problems.  Link to passing [build](https://ci.appveyor.com/project/blink1073/scikit-image/build/1.0.31).

### CODE REVIEW

**Comment by blink1073** [comment]
@stefanv, rebased.

**Comment by stefanv** [comment]
What I feared: https://ci.appveyor.com/project/blink1073/scikit-image/build/job/t7huuvgin48c7dft

**Comment by stefanv** [comment]
I can enable multi-status to include AppVeyor, but only once everything passes.

**Comment by blink1073** [comment]
Yay Microsoft!  Working on it now...

**Comment by blink1073** [comment]
Argh, `os.pathsep` is not `:` on Windows...

**Comment by stefanv** [comment]
Yikes!!

```
skimage.viewer.tests.test_widgets.test_ok_buttons ... alloc: invalid block: 05C98CF8: c0 5
```

**Comment by blink1073** [comment]
I'm travelling today so I may not get to this until tomorrow.

**Comment by blink1073** [comment]
That is a doozy.  I don't even know where to start...

**Comment by blink1073** [comment]
I might need to boot into Windows.

**Comment by blink1073** [comment]
Great, it works fine on my machine with and without PyQt installed....

**Comment by blink1073** [comment]
I used Windows 7 Anaconda Python 3.4 64 bit.

**Comment by blink1073** [comment]
Ready to roll. Not sure why coveralls is being mean.

**Comment by stefanv** [comment]
Most excellent; thank you @blink1073!

File: tools/appveyor/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Fetch numpy and scipy wheels from the sklearn rackspace wheelhouse.
+# Those wheels were generated by @ogrisel by calling `wheel convert` on
+# the binaries from http://www.lfd.uci.edu/~gohlke/pythonlibs/
+# This is a temporary solution. As soon as numpy, scipy and cython provide
+# official wheels for windows we can delete this --find-links line.
+--find-links http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/appveyor_pip_index.html
```
Comment (stefanv) [inline tools/appveyor/requirements.txt]: @matthew-brett Just wanted to confirm with you whether this is the most current situation?

**Pull request merged**